docs/self-managed/components/components-upgrade/870-to-880.md

```diff
@@ -752,3 +752,30 @@ Set one of the following:
 
 - YAML: `camunda.database.index.numberOfReplicas: 0`
 - Env var: `CAMUNDA_DATABASE_INDEX_NUMBER_OF_REPLICAS=0`
+
+## Web Modeler
+
+### Cluster configuration
+
+#### Removed authentication methods
+
+The previously deprecated authentication methods `OAUTH` and `CLIENT_CREDENTIALS` are no longer supported.
+If you still use one of them, update the [cluster configuration](/self-managed/components/modeler/web-modeler/configuration/configuration.md#clusters) in your Web Modeler installation:
+
+- Replace `OAUTH` with `BEARER_TOKEN` (the value was just renamed).
+- Instead of `CLIENT_CREDENTIALS` you can use `BEARER_TOKEN` authentication if the cluster is also on version 8.8 and uses [OIDC authentication](/self-managed/components/orchestration-cluster/identity/connect-external-identity-provider.md) with the same identity provider as Web Modeler.
```

Comment:
@jfriedenstab Isn't this a bit too strict? Web Modeler 8.8 is also able to communicate to clusters < 8.8 via bearer token auth, there's just the limitation of supporting multiple audiences in the connected identity provider.